console/src/components/Services/Data/TableInsertItem/InsertActions.js

```diff
@@ -28,17 +33,66 @@ const I_FETCH_ENUM_OPTIONS_ERROR = 'InsertItem/I_FETCH_ENUM_ERROR';
 const Open = () => ({ type: _OPEN });
 const Close = () => ({ type: _CLOSE });
 
+// insertItem helper
+const createInsertMigration = (
+  dispatch,
+  getState,
+  tableInfo,
+  insertedData,
+  primaryKeyInfo,
+  columns,
+  callback
+) => {
+  const upQuery = getInsertUpQuery(
+    { name: tableInfo.name, schema: tableInfo.schema },
+    insertedData
+  );
+  const downQuery = getInsertDownQuery(
+    { name: tableInfo.name, schema: tableInfo.schema },
+    insertedData,
+    primaryKeyInfo,
+    columns
+  );
+
+  const migrationName = `insert_into_${tableInfo.name}`;
+  const customOnSuccess = () => {
+    if (callback) {
+      callback();
+    }
```

Comment:
@beerose I was wondering if we should stack these notifications or maybe even add a timeout? So that the user has some time to view both the notifications.